README.md

```diff
@@ -4,29 +4,55 @@
 [![Build Status](https://travis-ci.org/jupyter/nbgrader.svg)](https://travis-ci.org/jupyter/nbgrader)
 [![codecov.io](http://codecov.io/github/jupyter/nbgrader/coverage.svg?branch=master)](http://codecov.io/github/jupyter/nbgrader?branch=master)
 
-A system for assigning and grading notebooks.
+A system for assigning and grading Jupyter notebooks.
 
 [Documentation can be found on Read the Docs.](http://nbgrader.readthedocs.org)
 
+## Visual highlights of nbgrader
+
+### Command line tools
+Command lines tools offer an efficient way for the user to generate, assign,
+release, collect, and grade notebooks.
+![nbgrader system and command line tools](docs/source/user_guide/images/read_only.png "nbgrader system and command line tools")
+
+### Toolbar extension for Jupyter notebooks
+The nbgrader toolbar extension for Jupyter notebooks guides the user to do
+assignment and grading tasks within the familiar Jupyter notebook interface.
+![nbgrader toolbar extension for Jupyter notebooks](docs/source/user_guide/images/assignment_toolbar.png "nbgrader toolbar extension for Jupyter notebooks")
+
+![nbgrader assignment list](docs/source/user_guide/images/assignment_list_downloaded_expanded.png "nbgrader assignment list")
```

Comment:
The gif is super cool. Can you make one for the command line tools? Perhaps two here: command line tools and toolbar extension both from the instructor perspective. Perhaps we should be specific in the docs and instead of using "user" use 'instructor' or 'student' where 'instructor'=='prof', 'ta', 'it support person'